editor/editor_interface.cpp

```diff
@@ -400,6 +400,21 @@ bool EditorInterface::is_movie_maker_enabled() const {
 }
 
 // Base.
+void EditorInterface::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	String pf = p_function;
+	if (p_idx == 0) {
+		if (pf == "set_main_screen_editor") {
+			for (String E : { "2D", "3D", "Script", "AssetLib" }) {
```

Comment:
I'd suggest avoiding manual quoting here, quoting isn't dynamic here, no need to save a few characters IMO